static/templates/message_body.hbs

```diff
@@ -2,14 +2,14 @@
     {{#unless status_message}}
     <span class="message_sender sender_info_hover no-select">
         {{#if include_sender}}
-
+        <span title="{{t 'User profile' }} (u)">
             {{> message_avatar}}
 
             <span class="sender_name auto-select" role="button" tabindex="0">{{msg/sender_full_name}}</span>
             {{#if sender_is_bot}}
             <i class="zulip-icon bot" aria-label="{{t 'Bot' }}"></i>
             {{/if}}
-
+        </span>
```

Comment:
Same here -- can't this just go on the `message_sender` span?

Reply:
The popup when triggered by hotkey (u) looks for `.sender_info_hover` and if we set the title attribute, that overrides the popover title set in JS, which in this case is the profile avatar. `.message_sender` also has the class `.sender_info_hover` which is why we cannot do this.

Reply:
Adding the attribute to the message_body only is working fine why are we adding it to me_message too?